spec/acceptance/validate_ruby_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'Running ruby validation' do
+describe 'pdk validate ruby', module_command: true do
```

Comment:
It is described in the main docs of rspec-core: https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-context#declare-a-shared-context-and-include-it-with-metadata